quickstart.md

```diff
@@ -759,14 +759,19 @@ Run skopeo to encrypt the image created in the previous section:
 
 ```
 sudo OCICRYPT_KEYPROVIDER_CONFIG=$(pwd)/attestation-agent/sample_keyprovider/src/enc_mods/offline_fs_kbs/ocicrypt.conf \
-skopeo copy --insecure-policy docker-daemon:[REGISTRY_URL]:unencrypted \
-docker-daemon:[REGISTRY_URL]:encrypted \
+skopeo copy --insecure-policy \
+docker-daemon:[REGISTRY_URL]:unencrypted \
```

Comment:
Can't we use `docker:` for the source as well?

Reply:
Yes, we can:

```
$ ./skopeo copy --insecure-policy --encryption-key provider:attestation-agent:${KEYS_FILE}:key_id1 docker://nginx:latest docker://wainersm/nginx-encrypted:test
Getting image source signatures
Copying blob d2c0556a17c5 done  
Copying blob 8740c948ffd4 done  
Copying blob b2fe3577faa4 done  
Copying blob c8b9881f2c6a done  
Copying blob 8316c5e80e6d done  
Copying blob 693c3ffa8f43 done  
Copying config 75eccb489a done  
Writing manifest to image destination
Storing signatures
$ ./skopeo inspect docker://wainersm/nginx-encrypted:test | grep MIMEType
            "MIMEType": "application/vnd.oci.image.layer.v1.tar+gzip+encrypted",
            "MIMEType": "application/vnd.oci.image.layer.v1.tar+gzip+encrypted",
            "MIMEType": "application/vnd.oci.image.layer.v1.tar+gzip+encrypted",
            "MIMEType": "application/vnd.oci.image.layer.v1.tar+gzip+encrypted",
            "MIMEType": "application/vnd.oci.image.layer.v1.tar+gzip+encrypted",
            "MIMEType": "application/vnd.oci.image.layer.v1.tar+gzip+encrypted",

```

Reply:
Updated.